Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -59,18 +59,6 @@ public enum EventKind: Int64, CaseIterable, Hashable {
     case notificationServiceRegistration = 6666
 }
 
-extension FetchedResults where Element == Event {
-    var unmuted: [Event] {
```

Comment:
filtering out muted events should be done in fetch requests, not on in-memory arrays of events. The NotificationsView was the last place this was being used, but it was unnecessary because the `Event.all(notifying:since:limit:)` filters them out.